azurelinuxagent/common/osutil/default.py

```diff
@@ -269,50 +255,58 @@ def remove_legacy_firewall_rule(self, dst_ip):
                 "Unable to remove legacy firewall rule, won't try removing it again. Error: {0}".format(ustr(error)))
 
     def enable_firewall(self, dst_ip, uid):
+        """
+        it's checks every iptable rule and add rules if not present. It's return tuple(enable firewall success status, update rules flag)
```

Comment:
NIT: typos - 

```suggestion
        It checks if every iptable rule exists and add them if not present. It returns a tuple(enable firewall success status, update rules flag)
```

Reply:
`We're anyways logging here (the multiple logger.info) so having telemetry doesn't make much difference`

Adding telemetry in osutil created circular import error. I guess I need to maintain two flags. Please let me know your thoughts, I don't want to undo and redo forever :)

Reply:
what's the circular import on? Why not just move the offending import to a separate common utility and fix the circular imports?